application_form/validators.py

```diff
@@ -48,3 +52,27 @@ def _calculate_control_character(self, individual_number: str) -> str:
         number = int(self.date_of_birth.strftime("%d%m%y") + individual_number)
         index = number % len(alphabet)
         return alphabet[index]
+
+
+class ProjectApplicantValidator:
+    def __call__(
+        self,
+        project_uuid: UUID,
+        date_of_birth_and_ssn_suffix: Union[Tuple[date, str], List[Tuple[date, str]]],
+    ):
+        if isinstance(date_of_birth_and_ssn_suffix, Tuple):
+            date_of_birth_and_ssn_suffix = [date_of_birth_and_ssn_suffix]
+        if isinstance(date_of_birth_and_ssn_suffix, List):
+            date_of_birth_and_ssn_suffix = date_of_birth_and_ssn_suffix.copy()
+
+        item = date_of_birth_and_ssn_suffix.pop()
```

Comment:
Before doing this I would recommend checking that the list is not empty as `.pop()` on an empty list throws an error.